src/main/scala/rocket/RocketCore.scala

```diff
@@ -418,6 +465,23 @@ class Rocket(tile: RocketTile)(implicit p: Parameters) extends CoreModule()(p)
     A2_IMM -> ex_imm,
     A2_SIZE -> Mux(ex_reg_rvc, 2.S, 4.S)))
 
+  val (ex_new_vl, ex_new_vconfig) = if (usingVector) {
+    val ex_avl = Mux(ex_ctrl.rxs1,
```

Comment:
what if `lmul != lmulMax`?

Reply:
https://github.com/chipsalliance/t1/blob/5022af424723b6c2828079f94155a86f9c8018a7/rocket/src/AbstractT1.scala#L223-L224

Reply:
You are right I think, will fix